Source/Prism/Mvvm/ErrorsContainer.cs

```diff
@@ -46,6 +46,15 @@ public bool HasErrors
             }
         }
 
+        /// <summary>
+        /// Returns all the errors in the container
+        /// </summary>
+        /// <returns>The dictionary of errors per property.</returns>
+        public Dictionary<string, List<T>> GetAllErrors()
```

Comment:
I'm curious if we should just call this `GetErrors()` like the other method, except without the proeprtyName parameter. @dansiegel what do you think? I don't feel strongly either way

Reply:
We currently have `ClearErrors(string)` and `ClearErrors()` which clears all errors, so it would make more sense to stick with this convention and name this `GetErrors()`

Reply:
@simedbn Can you rename this to simply `GetErrors()` and then we can get it merged.  Thanks.

Reply:
It's done Thanks